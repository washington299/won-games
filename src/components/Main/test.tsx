import { render } from "@testing-library/react";

import { Main } from ".";

describe("<Main />", () => {
	it("should render correctly", () => {
		const { container } = render(<Main />);

		expect(container.firstChild).toMatchSnapshot();
	});

	it("should render background-color correctly", () => {
		const { container } = render(<Main />);

		expect(container.firstChild).toHaveStyle({ "background-color": "#F00" });
	});
});
