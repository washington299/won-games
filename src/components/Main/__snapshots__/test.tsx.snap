// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render correctly 1`] = `
.c0 {
  background-color: #f00;
}

<div
  class="c0"
>
  <h1>
    title
  </h1>
  <p>
    description
  </p>
</div>
`;
